docs/architecture/state.md

```diff
@@ -18,70 +25,75 @@ The Miden Node(s) maintain three databases to describe the state:
 ![Architecture core concepts](../img/architecture/state/state.png){ width="80%" }
 </center>
 
+These databases are represented by authenticated data structures; for easily proving that items were added to or removed from a database, and so that a commitment to the database is very small.
```

Comment:
I stumbled upon this sentence, can you maybe rephrase to 


These databases are represented by authenticated data structures, enabling easy proof of items being added to or removed from a database and ensuring that the commitment to the database remains very small.